cmd/query.go

```diff
@@ -205,10 +209,11 @@ func queryUnrelayedPackets(ctx *config.Context) *cobra.Command {
 			}
 			out, err := json.Marshal(sp)
 			if err != nil {
+				logger.Error("failed to marshal sequences", zap.Any("sp", sp), zap.Error(err))
 				return err
 			}
 
-			fmt.Println(string(out))
+			logger.Info("unrelayed sequences", zap.String("sequences", string(out)))
```

Comment:
It seems that this print isn't a log output. Therefore you shouldn't change this in such a manner.

Reply:
Revert print